tools/export/exporters.py

```diff
@@ -114,14 +109,52 @@ def flags(self):
         flags['asm_flags'] += asm_defines
         flags['c_flags'] += c_defines
         flags['cxx_flags'] += c_defines
+        config_header = self.config_header_ref
         if config_header:
-            config_header = relpath(config_header,
-                                    self.resources.file_basepath[config_header])
-            flags['c_flags'] += self.toolchain.get_config_option(config_header)
+            flags['c_flags'] += self.toolchain.get_config_option(
+                config_header.name)
             flags['cxx_flags'] += self.toolchain.get_config_option(
-                config_header)
+                config_header.name)
         return flags
 
+    @property
```

Comment:
Interesting. Does this block of code now apply to more than just the gnuarmeclipse and nb exporters?

Reply:
Yes? I confess that github has prevented me from seeing the code in question.

Reply:
xD

This was in reference to moving `libraries()` and `toolchain_flags()` from those two exporters into this file.

Reply:
Yes.